packages/env/lib/env.js

```diff
@@ -247,8 +200,168 @@ module.exports = {
+ *
+ * @return {Config} The-env config object.
+ */
+async function initConfig() {
+	const configPath = path.resolve( '.wp-env.json' );
+	const config = await readConfig( configPath );
+
+	await fs.mkdir( config.workDirectoryPath, { recursive: true } );
+
+	await fs.writeFile(
+		config.dockerComposeConfigPath,
+		yaml.dump( buildDockerComposeConfig( config ) )
+	);
+
+	return config;
+}
+
+/**
+ * Copies a WordPress installation, taking care to ignore large directories
+ * (.git, node_modules) and configuration files (wp-config.php).
+ *
+ * @param {string} fromPath Path to the WordPress directory to copy.
+ * @param {string} toPath Destination path.
+ */
+async function copyCoreFiles( fromPath, toPath ) {
+	await copyDir( fromPath, toPath, {
+		filter( stat, filepath, filename ) {
+			if ( stat === 'symbolicLink' ) {
+				return false;
+			}
+			if ( stat === 'directory' && filename === '.git' ) {
+				return false;
+			}
+			if ( stat === 'directory' && filename === 'node_modules' ) {
+				return false;
+			}
+			if ( stat === 'file' && filename === 'wp-config.php' ) {
+				return false;
+			}
+			return true;
+		},
+	} );
+}
+
+/**
+ * Performs the given action again and again until it does not throw an error.
+ *
+ * @param {Function} action The action to perform.
+ * @param {Object} options
+ * @param {number} options.times How many times to try before giving up.
+ * @param {number} [options.delay=5000] How long, in milliseconds, to wait between each try.
+ */
+async function retry( action, { times, delay = 5000 } ) {
+	let tries = 0;
+	while ( true ) {
+		try {
+			return await action();
+		} catch ( error ) {
+			if ( ++tries >= times ) {
+				throw error;
+			}
+			await sleep( delay );
+		}
+	}
+}
+
+/**
+ * Checks a WordPress database connection. An error is thrown if the test is
+ * unsuccessful.
+ *
+ * @param {Config} config The wp-env config object.
+ */
+async function checkDatabaseConnection( { dockerComposeConfigPath } ) {
+	await dockerCompose.run( 'cli', 'wp db check', {
+		config: dockerComposeConfigPath,
+		commandOptions: [ '--rm' ],
+	} );
+}
+
+/**
+ * Configures WordPress for the given environment by installing WordPress,
+ * activating all plugins, and activating the first theme. These steps are
+ * performed sequentially so as to not overload the WordPress instance.
+ *
+ * @param {string} environment The environment to configure. Either 'development' or 'tests'.
+ * @param {Config} config The wp-env config object.
+ */
+async function configureWordPress( environment, config ) {
+	const options = {
+		config: config.dockerComposeConfigPath,
+		commandOptions: [ '--rm' ],
+	};
+
+	// Install WordPress.
+	await dockerCompose.run(
+		environment === 'development' ? 'cli' : 'tests-cli',
+		`wp core install
+			--url=localhost:${
+				environment === 'development'
+					? process.env.WP_ENV_PORT || '8888'
+					: process.env.WP_ENV_TESTS_PORT || '8889'
+			}
+			--title='${ config.name }'
```

Comment:
I believe the single quotes here are interpreted verbatim by WP-CLI and are causing the Site Title of the default installation to incorrectly include them:

![image](https://user-images.githubusercontent.com/1779930/75356432-f737b180-587d-11ea-8fbb-1a50d126ab42.png)

Depending whether it needs to be quoted to account for spaces in the name, it may need double quotes instead to work as expected?

```suggestion
			--title="${ config.name }"
```

```suggestion
			--title=${ config.name }
```

Reply:
Thanks! Fixed in https://github.com/WordPress/gutenberg/pull/20520.